tests/frameworks/OSXShims/include/windows.h

```diff
@@ -0,0 +1,55 @@
+//******************************************************************************
```

Comment:
We should standardize on whether we call this Windows or windows internally. OS X filesystems can be case-sensitive, and the include would be rightly confused.

You don't necessarily need to fix this here, but it will be something to remain aware of!